timeboost/src/lib.rs

```diff
@@ -130,6 +151,29 @@ impl Timeboost {
                         let e: CertifierDown = e;
                         return Err(e.into())
                     }
+                },
+                res = self.events.next() => match res {
+                    Some(comm_info) => {
+                        let cur: u64 = self.config.key_store.committee().id().into();
+                        let new_id: u64 = comm_info.id().into();
```

Comment:
```suggestion
                        let cur = self.config.key_store.committee().id();
                        let new_id = comm_info.id();
```
With the aforementioned `Add` and `Sub` impls for `CommitteeId` this would not be necessary.